cardstack/functions.php

```diff
@@ -1,4 +1,5 @@
 <?php
+header ("Access-Control-Allow-Origin:https://dev.animagia.pl/wp84591/");

```

Comment:
Use WordPress functions to get current site url (e.g. home_url), and use WordPress hooks to modify headers.